jsonschema-validation.md

```diff
@@ -748,39 +748,39 @@ associated schema.
 
 The value of this keyword MUST be a boolean. When multiple occurrences of this
 keyword are applicable to a single sub-instance, applications SHOULD consider
-the instance location to be deprecated if any occurrence specifies a true value.
+the instance location to be deprecated if any occurrence specifies a `true` value.
 
-If `deprecated` has a value of boolean true, it indicates that applications
+If `deprecated` has a value of boolean `true`, it indicates that applications
 SHOULD refrain from usage of the declared property. It MAY mean the property is
 going to be removed in the future.
 
-A root schema containing `deprecated` with a value of true indicates that the
+A root schema containing `deprecated` with a value of `true` indicates that the
 entire resource being described MAY be removed in the future.
 
 The `deprecated` keyword applies to each instance location to which the schema
 object containing the keyword successfully applies. This can result in scenarios
 where every array item or object property is deprecated even though the
 containing array or object is not.
 
-Omitting this keyword has the same behavior as a value of false.
+Omitting this keyword has the same behavior as a value of `false`.
 
 ### `readOnly` and `writeOnly`
 
 The value of these keywords MUST be a boolean. When multiple occurrences of
 these keywords are applicable to a single sub-instance, the resulting behavior
-SHOULD be as for a true value if any occurrence specifies a true value, and
-SHOULD be as for a false value otherwise.
+SHOULD be as for a `true` value if any occurrence specifies a `true` value, and
+SHOULD be as for a `false` value otherwise.
 
-If `readOnly` has a value of boolean true, it indicates that the value of the
+If `readOnly` has a value of boolean `true`, it indicates that the value of the
 instance is managed exclusively by the owning authority, and attempts by an
-application to modify the value of this property are expected to be ignored or
+application to modify the value of an instance are expected to be ignored or
```

Comment:
When I read "an instance" my mind went to the "the entire JSON document", which isn't the intention. I think "instance" can be used to refer to a fragment of a JSON document, but we usually don't use it that way so this might be confusing.

What if we just leave it at "the value" and drop the "of" part entirely? Or we could say "that value within an instance"?

Reply:
or "the local instance"

Reply:
how about : 
```
If `readOnly` has a value of boolean `true`, it indicates that the
corresponding value in the instance is
managed exclusively by the owning authority, and attempts by an
application to modify the value are expected to be ignored or
rejected by that owning authority.
```

Reply:
Got that from https://www.learnjsonschema.com/2020-12/meta-data/readonly/